zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/ZebedeeReader.java

```diff
@@ -120,7 +121,11 @@ public Content getCollectionContent(String collectionId, String sessionId, Strin
      * @throws IOException
      */
     public Resource getPublishedResource(String path) throws ZebedeeException, IOException {
-        return publishedContentReader.getResource(path);
+        try {
+            return publishedContentReader.getResource(path);
+        } catch (BadRequestException e) {
+            return publishedContentReader.getResource(path + "/index.html");
```

Comment:
Same here to we want to only apply this to uris starting /visualisation? I know that index.html is a convention but its not one this code follows. I am wary of applying this to everything.